main/src/ca/uwaterloo/flix/language/ast/LiftedAst.scala

```diff
@@ -29,7 +29,10 @@ object LiftedAst {
                   entryPoint: Option[Symbol.DefnSym],
                   sources: Map[Source, SourceLocation])
 
-  case class Def(ann: Ast.Annotations, mod: Ast.Modifiers, sym: Symbol.DefnSym, fparams: List[LiftedAst.FormalParam], exp: LiftedAst.Expression, tpe: Type, loc: SourceLocation)
+  // Explicit parameters (x below) are kept separate from captured parameters
+  // (a below) for the sake of their JVM representation
+  // x -> x + a
```

Comment:
I find this comment to be too cryptic. I would either improve it (a lot) or remove it. Probably remove it.